CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # `chainweb-node` Changelog
 
+## 2.22 (2023-11-XX)
+ This version replaces all previous versions. Any prior version will stop working
+ on **2023-12-13T:00:00Z**. Node administrators must upgrade to this version before
+ that date.
+ 
+ This version will expire on **2023-03-06T:00:00Z**.
+ 
+ To upgrade, pull the latest docker image or download the binary and restart the node.
+ 
+ Changes:
+ 
+Pact support for webauthn keys (#1779)
```

Comment:
I'd amend this to say "node support for webauthn signers, scoped signatures, and webauthn keyset formats in Pact"